chia/util/streamable.py

```diff
@@ -116,7 +134,22 @@ def dataclass_from_dict(klass, d):
         return klass(d)
 
 
-def recurse_jsonify(d):
+@overload
+def recurse_jsonify(d: List[Any]) -> List[Any]:
+    ...
+
+
+@overload
+def recurse_jsonify(d: Tuple[Any, ...]) -> Tuple[Any, ...]:
+    ...
+
+
+@overload
+def recurse_jsonify(d: Dict[str, Any]) -> Dict[str, Any]:
+    ...
+
+
+def recurse_jsonify(d):  # type: ignore[no-untyped-def]
```

Comment:
Others agree about the excessive whitespace.  https://github.com/psf/black/issues/1797

Without saying this is a beautiful situation...  here's a thing that I would start with and see if mypy is ok with it.

```suggestion
def recurse_jsonify(d: Union[List[Any], Tuple[Any, ...], Dict[str, Any]]) -> Union[List[Any], Dict[str, Any]:
```

Reply:
Also missing a `]` :) See 5e2ba5f6060161e04cb73365e242e63b774452d5.